recipes/nitime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.6" %}
+
+package:
+    name: nitime
+    version: {{ version }}
+
+source:
+    fn: nitime-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/n/nitime/nitime-{{ version }}.tar.gz
+    md5: 43d92863b8792a50336b40635999f909
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - numpy
```

Comment:
It is and I need to fix it.

https://github.com/nipy/nitime/blob/master/setup.py#L40